prisma/schema.prisma

```diff
@@ -32,12 +32,41 @@ model Team {
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
 
+  competitionTeams CompetitionTeams[]
+
+  @@map("competitions")
+}
+
+model Team {
```

Comment:
removing the relation with sport has also broken our get teams route `src/app/api/teams/route.ts`

i know we are refactoring this to select a competition before team, but just so the app isn't crashing lets just remove the where clause in that file so we fetch all teams in the db

Reply:
yep sweet - done